packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -231,6 +293,39 @@ export class CfnInclude extends core.CfnElement {
     this.conditions[conditionName] = cfnCondition;
   }
 
+  private createNestedStack(nestedStackId: string, props?: CfnIncludeProps): void {
+    if (nestedStackId in this.template.Resources) {
+      if (this.template.Resources[nestedStackId].Type === 'AWS::CloudFormation::Stack' && props?.nestedStacks) {
+        const nestedStackScope = new core.NestedStack(this, 'Nested' + nestedStackId, {
+          parameters: this.template.Resources[nestedStackId].Properties.Parameters,
+          notificationArns: this.template.Resources[nestedStackId].Properties.NotificationArns,
+          timeout: this.template.Resources[nestedStackId].Properties.Timeout,
```

Comment:
I believe we're not testing either of these properties (`NotificationArns` and `Timeout` anywhere).

Add them to one of the tested templates.

Reply:
Actually, I just realized.

This is not handling any resource attribute of `CfnStack` other than `Properties` (things like`Metadata`, creation/update policies, `Condition`, etc.).

All of that needs to be added, and covered in tests as well.

Reply:
You sure that we need other attributes? `NestedStackProps` only take the `Properties` attributes.

Reply:
Notice I said "Attributes", not "Properties" 🙂 https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-product-attribute-reference.html